registry/repositories.go

```diff
@@ -4,7 +4,7 @@ type repositoriesResponse struct {
 	Repositories []string `json:"repositories"`
 }
 
-func (registry *Registry) Repositories() (repos []string, err error) {
```

Comment:
I definitely didn't mean to include that. I missed it in the large number of files updated. Good catch!
I've removed it.